### PR TITLE
Teleop Fix

### PR DESCRIPTION
**Pull request by chamorajg**
-  Fixes the teleop working for stompy
-  Fixed motion retargetting 
-  Clean unnecessary comments.
-  Modify the velocity parameters to help it achieve reference motion.

### CODE REVIEW

File: phc/phc/utils/torch_stompy_humanoid_batch.py [hunk 1]
```diff
@@ -46,7 +45,7 @@ def __init__(self, mjcf_file = f"legged_gym/resources/robots/stompypro/robot_tes
         self.extend_head = extend_head
         if extend_hand:
             self.model_names = mjcf_data['node_names'] + ["left_hand_keypoint_link", "right_hand_keypoint_link"]
-            self._parents = torch.cat((mjcf_data['parent_indices'], torch.tensor([13, 17]))).to(device) # Adding the hands joints
+            self._parents = torch.cat((mjcf_data['parent_indices'], torch.tensor([9, 18]))).to(device) # Adding the hands joints
```
Comment (budzianowski) [inline phc/phc/utils/torch_stompy_humanoid_batch.py]: make it a variable, not random indices

File: save.txt [hunk 2]
```diff
@@ -0,0 +1,220 @@
+absl-py==2.1.0
```
Comment (budzianowski) [inline save.txt]: rename to requirements_saved.txt

File: legged_gym/resources/robots/stompypro/robot_test.urdf [hunk 3]
```diff
@@ -23,7 +23,7 @@
     <parent link="torso_link" />
```
Comment (budzianowski) [inline legged_gym/resources/robots/stompypro/robot_test.urdf:23]: keep urdf definition and reuse it across the scripts

**Review by budzianowski — APPROVED** [review]
the efforts and velocities make more sense now! let's merge it.

**Comment by chamorajg** [comment]
Pushed the code changes as requested. Merging the branch.

**Pull request merged**